packages/void-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS builder
+WORKDIR /app
+
+ENV BUILD_PLAYGROUND=1
+
+# Build the package with the BUILD_PLAYGROUND flag set
+RUN pnpm --filter @scalar/void-server build
+
+FROM node:20-bullseye-slim AS runner
+COPY --from=builder /usr/bin/dumb-init /usr/bin/dumb-init
+
+ENV NODE_ENV=production
+ENV PORT=$PORT
+ENV HOST='0.0.0.0'
```

Comment:
@tmastrom What are we using these for? $PORT is already exposed and I don't see us having a need to change the host?

Reply:
These aren't necessary